client/landing/jetpack-cloud/components/activity-card/index.jsx

```diff
@@ -19,13 +20,18 @@ import './style.scss';
 
 class ActivityCard extends Component {
 	render() {
-		const { activity, moment, allowRestore } = this.props;
+		const { activity, allowRestore, timezone, gmtOffset } = this.props;
+
+		const backupTimeDisplay = applySiteOffset( activity.activityTs, {
+			timezone,
+			gmtOffset,
+		} ).format( 'H:mm' );
```

Comment:
I noticed in the previous commit we were returning a string formatted with `'LT'`, which seems like it's locale-aware. Do we want to continue that here, or is `'H:mm'` an intentional choice?

```suggestion
		} ).format( 'LT' );
```

Reply:
Hey @mattgawarecki, thank you to review it :) Yes, it's intentional, a few days ago I asked Yvonne if at the end the time format would be 0-23, and she confirmed it. Yes, I know that the design doesn't match

Reply:
@mattgawarecki, in the end, we are going to use the locale format LT, so I'm going to apply the change.